### PR TITLE
docs: improved SEO & fixed typo and localization issues 

### PR DESCRIPTION
**Pull request by Neilblaze**
### Description
Optimizing _keywords_ and _metadata_ of the docs enhances the _SEO_ of APISIX & overall increases the accessibility for current and potential users. Hence, a few of the docs having "_APISIX_" has been updated to "_**Apache APISIX**_" (like the rest), and the keyword "API Gateway" or "API 网关" (in ZH) has been synchronously added to the list. 

Moreover, a few typos and localization mistakes detected during the process have also been rectified.

Fixes: #7218 
Refer to: #8966 

cc: @juzhiyuan, @nfrankel, @navendu-pottekkat, @Ajay-singh1

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/opentelemetry.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ---
 title: opentelemetry
+keywords:
+  - Apache APISIX
+  - API 网关
+  - Plugin
+  - OpenTelemetry
+  - opentelemetry
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/opentelemetry.md:8]: Keep one item is ok

```suggestion
  - opentelemetry
```

**Review by juzhiyuan — COMMENTED** [review]
LGTM.

P.S. From my understanding, Google and other Search Engines don't rank pages according to keywords fields anymore 😅 but this PR does enhance the Apache APISIX branding.

**Review by Ajay-singh1 — CHANGES_REQUESTED** [review]
@Neilblaze Do some changes as mentioned by @juzhiyuan.

**Review by navendu-pottekkat — APPROVED** [review]
LGTM after @juzhiyuan 's comment

**Comment by Neilblaze** [comment]
Damn, it's so annoying! :(
Apparently, GitHub web doesn't allow to trigger multi-review request at once lol

**Comment by Neilblaze** [comment]
@Ajay-singh1 this is a low-priority PR. Reviewers are already tagged & they'll review it accordingly. Hence refrain from tagging them again and again!

**Comment by Ajay-singh1** [comment]
@Neilblaze Ok!

**Review by moonming — APPROVED** [review]

**Review by navendu-pottekkat — APPROVED** [review]

**Pull request merged**